module/VuFind/src/VuFind/AjaxHandler/GetItemStatuses.php

```diff
@@ -571,7 +571,7 @@ public function handleRequest(Params $params)
         foreach ($missingIds as $missingId => $recordNumber) {
             $availabilityStatus = $this->availabilityStatusManager->createAvailabilityStatus(false);
             $statuses[] = [
-                'id'                   => $missingId,
+                'id'                   => strval($missingId),
```

Comment:
Any reason not to use a typecast here?


```suggestion
                'id'                   => (string)$missingId,
```

The build is failing because it's complaining about a missing declaration for `strval`, but maybe it's better not to use `strval` at all.